linter.py

```diff
@@ -22,7 +22,7 @@
 
 MYPY = False
 if MYPY:
-    from typing import List, Union
+    from typing import List, Union  # noqa: F401
```

Comment:
This is introduced by a breaking change in flake8.  I just pinned their version so this `noqa` can be removed after rebasing onto the master.